#detection_data.msg
#topic:/detection/data
uint8 detection_id
float32 detection_x
float32 detection_y